lib/src/main/java/graphql/nadel/NextgenEngine.kt

```diff
@@ -188,6 +192,7 @@ class NextgenEngine @JvmOverloads constructor(
         topLevelField: ExecutableNormalizedField,
         service: Service,
         executionContext: NadelExecutionContext,
+        instrumentationState: InstrumentationState?,
```

Comment:
Can we pull the `InstrumentationState` out of `NadelExecutionContext` instead of passing in as another parameter?